### DISCUSSION TITLE
Maintenance rights for executablebooks/github-activity ?

### DISCUSSION THREAD

**Discussion by consideRatio** [executablebooks/meta#355] [General] [answered]
Hi!

I'm caring of the executablebooks/github-activity project and just created a PR to update a PR. I wonder if I could could get maintainer rights there? It is a low maintenance project in general currently and was imported from chrisholdgraf/github-activity into the org.

Ping @choldgraf

**Comment by choldgraf** [comment, marked as answer]
Done 👍

**Reply by consideRatio** [reply]
Thanks!
